src/rules/max-statements.md

```diff
@@ -134,9 +133,9 @@ let foo = () => {
 
 :::
 
-Note that this rule does not apply to class static blocks, and that statements in class static blocks do not count as statements in the enclosing function.
+注意，这条规则不适用于类静态块，而且类静态块中的语句不算是包围函数的语句。
 
-Examples of **correct** code for this rule with `{ "max": 2 }` option:
+本规则的**正确的**代码的例子，有`{ "max": 2 }`选项的情况下，本规则的**正确代码示例。
```

Comment:
https://deploy-preview-72--zh-hans-docs.netlify.app/rules/max-statements#ignoretoplevelfunctions
Seems like there are either ** missing or ** additionally present and breaks in render
```suggestion
本规则的**正确的**代码的例子，有`{ "max": 2 }`选项的情况下，本规则的正确代码示例。
```